docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Coverflow.htm

```diff
@@ -0,0 +1,224 @@
+
+<p>Elements with the <code>class=&quot;ui-coverflow&quot;</code> attribute can be created with Coverflow.<br>
+Changing the value of <code>data-effect=&quot;coverflow&quot;</code> applies a variety of effects.<br>
+Coverflow images are defined inside <code>&lt;ul class=&quot;flip-items&quot;&gt;</code>.<br>
+Images are recognized as coverflow when declared with the <code>&lt;li&gt;</code> tag.</p>
+
+<h2><a id="html-examples-coverflow-mobile"></a>HTML Examples</h2>
+
+<ul>
+
+<li>
+
+<p><a id='create element-coverflow-mobile'></a>To create a coverflow using the <code>&lt;ui-coverflow&gt;</code> element:</p>
+
+<pre class="prettyprint">
+    &lt;div class=&quot;ui-coverflow&quot; data-effect=&quot;coverflow&quot;&gt;
+    &lt;ul class=&quot;flip-items&quot;&gt;
+        &lt;li&gt;&lt;img src=&quot;sample1.png&quot;&gt;&lt;/li&gt;
+        &lt;li&gt;&lt;img src=&quot;sample2.png&quot;&gt;&lt;/li&gt;
+        &lt;li&gt;&lt;img src=&quot;sample3.png&quot;&gt;&lt;/li&gt;
+    &lt;/ul&gt;
+</pre>
+
+</li>
+
+</ul>
+
+<h2><a id="options-for widget-coverflow-mobile"></a>Options</h2>
+
+<p>The options for a component can be defined as <code>data-...</code> attributes or passed as parameters to the constructor.</p>
+
+<p>You can change an option for the component using the <code>option</code> method.</p>
+
+	<h3>Summary</h3>
+
+	<table>
+		<tbody>
+		<tr>
+			<th>Option</th>
+			<th>Input type</th>
+			<th>Default value</th>
+			<th>Description</th>
+		</tr>
+
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">data-effect</span></td>
+				<td class="option">string | null</td>
+				<td class="option">"coverflow"</td>
+				<td class="option">Effect type.</td>
+			</tr>
+
+		</tbody>
+	</table>
+
+
+
+<h2><a id="methods-coverflow-mobile"></a>Methods</h2>
+
+<p>To call a method on the component, use one of the existing APIs:</p>
+
+<pre class="prettyprint">&lt;div class=&quot;ui-coverflow&quot;&gt;&lt;/div&gt;
+&lt;script&gt;
+    var coverflowElement = document.getElementsByClassName(&quot;ui-coverflow&quot;),
+        coverflow = tau.widget.CoverFlow(coverflowElement);
+
+    coverflow.value(&quot;carousel&quot;);
+&lt;/script&gt;
+</pre>
+
+	<h3>Summary</h3>
+	<table class="informaltable">
+		<thead>
+		<tr>
+			<th>Method</th>
+			<th>Description</th>
+		</tr>
+		</thead>
+		<tbody>
+
+
+				<tr>
+					<td>
+						<pre class="intable prettyprint">string <a href="#method-value">value</a>(string? value) </pre>
+					</td>
+					<td><p>Gets or sets a value.</p></td>
+				</tr>
+
+
+		</tbody>
+	</table>
+	<dl>
+
+
+
+				<dt class="method" id="addidp28072"><code><b><span class="methodName"
+																   id="method-value">value</span></b></code></dt>
+				<dd>
+					<div class="brief">
+						<p>Gets or sets a value.</p>
+					</div>
+					<div class="synopsis">
+						<pre class="signature prettyprint">string value(string? value) </pre>
+					</div>
+
+						<p><span class="version">Since: </span>
```

Comment:
Would you please check this "**Since:** 2.3"?

Reply:
5.5 correct. fixed.

done.